.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+Thank you for your contribution!
+
+Please provide a summary of the changes and the motivation behind the change.
+
+## Reviewer Checklist
+
+- [ ] Documentation
+- [ ] Unit Tests
+- [ ] E2E Tests
+- [ ] Commits are prefixed with one of [`Bug Fix:` , `Feature:`, `Doc:`]
```

Comment:
Do we _really_ care about commit prefixes at this point? Do we have automation already that:
- enforces this
- uses it for something (i assume the purpose is to build changelogs)

Does this mean we're ready to start forcing PRs to do just one of thses things?

Reply:
+1 on removing it commit prefix stuff from existing milestone, IMO.

Reply:
Removed.